app/models/pipeline_run_stage.rb

```diff
@@ -226,12 +171,11 @@ def host_filtering_command
     attribute_dict[:fastq2] = sample.input_files[1].name if sample.input_files[1]
     dag_commands = prepare_dag("host_filter", attribute_dict)
 
-    upload_version = "idseq_dag --version | cut -f2 -d ' ' | aws s3 cp  - #{pipeline_run.pipeline_version_file} "
-    batch_command = [install_pipeline, upload_version, dag_commands].join("; ")
+    batch_command = [install_pipeline(pipeline_run.pipeline_commit), upload_version(pipeline_run.pipeline_version_file), dag_commands].join("; ")
 
     # Dispatch job
     memory = sample.sample_memory.present? ? sample.sample_memory : Sample::DEFAULT_MEMORY_IN_MB
-    aegea_batch_submit_command(batch_command, memory)
+    aegea_batch_submit_command(batch_command, memory, job_queue: pipeline_run.sample.job_queue)
```

Comment:
This change in particular looks like it might affect the batch queue to which host filtering jobs are sent.   Can you confirm those jobs would still go to the idseq_himem queue?   What need is, quite simply, host filtering and alignment viz on idseq_himem, and non-host-alignment on idseq.   Non-host-alignment should share queue with anything else.   Ideally host filtering would also not share queues with anything else, so some day we should create additional queues to support that.   Also add autoscaling for those.

Reply:
resolved